client/html/pages/panel/profile.html

```diff
@@ -39,95 +53,161 @@
-                        <span class="text-lg text-light">Web Design Studios</span>
-                      </p>
-                      <p>
-                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc vehicula, magna et bibendum malesuada, purus augue suscipit dolor, vitae fringilla dui nibh non lectus. Curabitur in pellentesque tortor. Nunc posuere vestibulum augue, quis posuere orci blandit vitae. Suspendisse dignissim elit dui, ac dictum felis interdum nec.
-                      </p>
-                    </div>
-                  </div>
-                </div>
-              </li>
-              <li>
-                <div class="timeline-circ style-accent"></div>
-                <div class="timeline-entry">
-                  <div class="card style-default-bright">
-                    <div class="card-body small-padding">
-                      <small class="text-uppercase text-primary pull-right">Juli, 2008 - January, 2014</small>
-                      <p>
-                        <span class="text-lg text-medium">Commercial manager</span><br/>
-                        <span class="text-lg text-light">Creation Inc.</span>
-                      </p>
-                      <p>
-                        Nunc vehicula, magna et bibendum malesuada, purus augue suscipit dolor, vitae fringilla dui nibh non lectus. Curabitur in pellentesque tortor. Nunc posuere vestibulum augue, quis posuere orci blandit vitae. Suspendisse dignissim elit dui, ac dictum felis interdum nec.
-                      </p>
-                    </div>
-                  </div>
-                </div>
-              </li>
-              <li>
-                <div class="timeline-circ style-accent"></div>
-                <div class="timeline-entry">
-                  <div class="card style-default-bright">
-                    <div class="card-body small-padding">
-                      <small class="text-uppercase text-primary pull-right">March, 2002 - Juli, 2008</small>
-                      <p>
-                        <span class="text-lg text-medium">Commercial assistant</span><br/>
-                        <span class="text-lg text-light">Designers Corp.</span>
-                      </p>
-                      <p>
-                        Suspendisse dignissim elit dui, ac dictum felis interdum nec. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc vehicula, magna et bibendum malesuada, purus augue suscipit dolor, vitae fringilla dui nibh non lectus. Curabitur in pellentesque tortor. Nunc posuere vestibulum augue, quis posuere orci blandit vitae.
-                      </p>
+            <div class="row">
+              <div class="col-sm-9 col-xs-12">
+                <h3 class="opacity-50">Sobre mim</h3>
+                {{#if user.profile.aboutme.length}}
+                  {{{user.profile.aboutme}}}
+                {{else}}
+                  <p>Preencha esse campo na área "Resumo Profissional" em 
```

Comment:
I think this way we are showing this message to all users, the account owner and all the others that might see the user's profile. We should take care not to show this to everyone.